docs/config/README.md

```diff
@@ -2561,7 +2561,7 @@ If you have an interesting example not covered there, feel free to share it ther
 | Option        | Default                       | Description                                                                                                                |
 | ------------- | ------------------------------- | -------------------------------------------------------------------------------------------------------------------------- |
 | `command`     |                                 | The command whose output should be printed. The command will be passed on stdin to the shell.                              |
-| `when`        |                                 | A shell command used as a condition to show the module. The module will be shown if the command returns a `0` status code. |
+| `when`        | `false`                         | Either a boolean value (`true` or `false`) or a string shell command used as a condition to show the module. In case of a string, the module will be shown if the command returns a `0` status code. |
```

Comment:
@davidkna, do you think adding _"(`true` or `false`, without quotes)_" would make it more obvious, especially in regards to the comment about `"true"` would still run the command?